mesmer/mesmer_m/harmonic_model.py

```diff
@@ -12,6 +12,7 @@
 import xarray as xr
 
 import mesmer
+from mesmer.core.utils import _check_dataarray_form
 
 
 def generate_fourier_series_np(yearly_T, coeffs, months):
```

Comment:
```suggestion
def _generate_fourier_series_np(yearly_T, coeffs, months):
```

Is probably easier to do this in vscode to get all the necessary changes also in the tests etc.